Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: rand(30000)+1234
```

Comment:
:-1: for doing random hack like that.

As I remember if `2222` is used, Vagrant will try to use `2200` instead and so on.

@humblearner what `Vagrant` version do you have?